ci/cpu/upload.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+#
+# Adopted from https://github.com/tmcdonell/travis-scripts/blob/dfaac280ac2082cd6bcaba3217428347899f2975/update-accelerate-buildbot.sh
+
+set -e
+
+# Setup 'gpuci_retry' for upload retries (results in 4 total attempts)
+export GPUCI_RETRY_MAX=3
+export GPUCI_RETRY_SLEEP=30
+
+# Set default label options if they are not defined elsewhere
+export LABEL_OPTION=${LABEL_OPTION:-"--label main"}
+
+# Skip uploads unless BUILD_MODE == "branch"
+if [ ${BUILD_MODE} != "branch" ]; then
+  echo "Skipping upload"
+  return 0
+fi
+
+# Skip uploads if there is no upload key
+if [ -z "$MY_UPLOAD_KEY" ]; then
+  echo "No upload key"
+  return 0
+fi
+
+################################################################################
+# SETUP - Get conda file output locations
+################################################################################
+
+gpuci_logger "Get conda file output locations"
+
+export LIBCLX_FILE=`conda build conda/recipes/libclx --output`
+export CLX_FILE=`conda build conda/recipes/clx --python=$PYTHON --output`
+
+################################################################################
+# UPLOAD - Conda packages
+################################################################################
+
+gpuci_logger "Starting conda uploads"
+
+if [ "$BUILD_LIBCLX" == "1" && "$UPLOAD_LIBCLX" == "1" ]; then
```

Comment:
This needs to be surrounded by `[[ ]]` due to the use of the &&

Same for the one below

Reply:
Hm, I copy pasted this file from other PRs, so the syntax should be consistent, and it worked for the others. Problem might lie elsewhere, I'll check.